tests/integration/components/polaris-data-table-test.js

```diff
@@ -0,0 +1,149 @@
+import { helper } from '@ember/component/helper';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { findAll, find } from 'ember-native-dom-helpers';
```

Comment:
we should use the new find, findAll from `@ember/test-helpers`....we can probably start deprecating native-dom helpers

Reply:
not saying this should not go out because of that^

we should also use qunit-dom for assertion, from what I've seen much nicer error messages